<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pregame_menu</class>
 <widget class="QWidget" name="pregame_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1206</width>
    <height>800</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Слепопечать v0.1</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#game_menu {
	background-color: transparent;
	background: none;
	border: none;
	background-repeat: none;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1">
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="main_layout" stretch="3,1,1,3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <property name="topMargin">
      <number>30</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>25</number>
     </property>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="pregame_logo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>250</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>650</width>
         <height>300</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#pregame_logo {
background-color: transparent;
border-image: url(:/images/images/pregame_logo.png);
background: none;
border: none;
background-repeat: none;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="buttons_layout"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="name_label">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>85</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>110</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>23</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Ваше имя: KIRILL</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lang_label">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>75</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>23</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Язык текста: RU</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="level_label">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>75</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>110</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>23</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Уровень: ва ол</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="startgame_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>325</width>
         <height>75</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>325</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#startgame_button {
    background: transparent;
    font: &quot;Comic Sans MS&quot;;
    font-size: 30pt;
}

#startgame_button:pressed {
    font: &quot;Comic Sans MS&quot;;
    font-size: 28pt;
}</string>
       </property>
       <property name="text">
        <string>Вперёд!</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item alignment="Qt::AlignRight">
      <widget class="QPushButton" name="pregame_quit_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#pregame_quit_button {
background-color: transparent;
border-image: url(:/icons/images/widget_quit.png);
background: none;
border: none;
background-repeat: none;
}
#pregame_quit_button:pressed
{
   border-image: url(:/icons/images/widget_quit_clicked.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="shortcut">
        <string>Esc</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qrc_files/pictures.qrc"/>
 </resources>
 <connections/>
</ui>
